pages/common/gnomon.md

```diff
@@ -0,0 +1,11 @@
+# gnomon 
+
+> Utility to annotate console logging statements with timestamps and find slow processes.
+
+- Use UNIX (or DOS) pipes to pipe the stdout of any command through gnomon:
+
+`npm test | gnomon`
+
+- Use command-line options to adjust gnomon's behavior for any command:
```

Comment:
This example seems rather general. The point of a tldr-page is to demonstrate the most useful arguments and combinations so that you don't have to look them up :-)

Reply:
Maybe it's enough `npm test | gnomon` command?

Reply:
Looking at the README, there are a bunch of potentially useful flags there. We could cover `--type=elapsed-total`, `--high` & `--medium`, and even `--real-time=false` (not all in the same example, of course :P

Reply:
Looking back, I don't think I explained myself very well. It's the `Use command-line options` in this example that I'm unsure about, as it implies to me that I need to read the man page - and that's what tldr-pages should help you have to do less :P